src/fragments/start/getting-started/android/setup.mdx

```diff
@@ -52,6 +52,9 @@ Amplify for Android is distributed as an Apache Maven package. In this section,
    dependencies {
        implementation 'com.amplifyframework:aws-api:ANDROID_VERSION'
        implementation 'com.amplifyframework:aws-datastore:ANDROID_VERSION'
+       def lifecycle_version = "2.4.0"
+       implementation "androidx.lifecycle:lifecycle-viewmodel:$lifecycle_version"
+       implementation "androidx.lifecycle:lifecycle-viewmodel-ktx:$lifecycle_version"
```

Comment:
IMO putting the lifecycle version inside the libraries would be more consistent. People can create the version def at some other place like they do in their projects. 

In the Android Codelabs they use the approach below only when they define the root project external version numbers [here](https://developer.android.com/codelabs/android-room-with-a-view?hl=en#3):
`implementation "androidx.lifecycle:lifecycle-viewmodel:$rootProject.lifecycleVersion"`

```suggestion
       implementation "androidx.lifecycle:lifecycle-viewmodel:2.4.0"
       implementation "androidx.lifecycle:lifecycle-viewmodel-ktx:2.4.0"
```